### ISSUE TITLE
Dynamic return url

### ISSUE THREAD

**Issue by Glutch** [liamcurry/passport-steam#102]
I have a login button on multiple pages, and i want the user to be returned to the page they logged in from. Not always '/'.

How can i solve this?

**Comment by DoseOfCode** [comment]
Set the redirect url you want the user to go to in a session, then once they come back from Steam logged in you can redirect them to it.

**Comment by Marcus-Cardoso** [comment]
@Glutch Did you manage to do this?

**Comment by Glutch** [comment]
> @Glutch Did you manage to do this?

Yes i believe so, but it was such a long time ago and the project is abandoned so i can't find it.
Keep searching and you will figure it out
